src/handlers/localstorage/globalSettings.ts

```diff
@@ -32,9 +32,9 @@ export const getLanguage = () => getGlobal(LANGUAGE, Language.EN_US);
 
 export const saveLanguage = (language: any) => saveGlobal(LANGUAGE, language);
 
-export const getNetwork = () => getGlobal(NETWORK, networkTypes.mainnet);
+export const getChainId = () => getGlobal(CHAIN_ID, ChainId.mainnet);
```

Comment:
confirming we don't need a migration for this as everyone should be on `mainnet` for the global setting (mainnet vs testnet which we currently do not support) anyway

Reply:
ok i'm gonna add a fallback to the last local network state just to be safe